### PR TITLE
Add middleware for adding CORS headers

### PR DESCRIPTION
**Pull request by xendk**
DDBLM-40

### CODE REVIEW

File: bootstrap/app.php [hunk 1]
```diff
@@ -55,6 +55,10 @@
 |
 */
 
+$app->middleware([
+    App\Http\Middleware\CorsMiddleware::class,
+]);
+
 // $app->routeMiddleware([
 //     'auth' => App\Http\Middleware\Authenticate::class,
 // ]);
```
Comment (lasseborly) [inline bootstrap/app.php]: Huh?

Comment (xendk) [reply bootstrap/app.php]: Huh what?

Comment (kasperg) [reply bootstrap/app.php]: We originally kept the commented code as an example of how to add middleware. If we actually add a middleware the example is no longer needed and should be removed.

**Comment by xendk** [comment]
Do we think we ever need to configure CORS, or do we think we'll be sticking with headers that allows all?

I believe the latter, and don't see much value in including a package which primarily adds configurability. If it changes in the future, we can swap them around.

File: bootstrap/app.php (hunk 1 above)
Comment (xendk) [reply bootstrap/app.php]: Different kind of middleware...

**Comment by kasperg** [comment]
The primary value I see is in reusability. At the moment we have two services which have the exact same technical need. There is a relatively simple community solution from a well-etablished vendor with a record of fast updates so why not use it?

I agree that we have no need for configurability. I would think that the initial configuration would be the same as what you implemented here.

I could not spot any recent package which provided the same functionality with the default configuration matching our needs. We could provide our own package which contained the implementation but that quickly becomes tedious.

File: bootstrap/app.php (hunk 1 above)
Comment (lasseborly) [reply bootstrap/app.php]: Can we comment... the comment saying that this is an example and therefore we are keeping it? Just to give some context and so it doesn't seem so scary?

**Comment by xendk** [comment]
#40

**Pull request closed without merging**